src/index.e2e.test.js

```diff
@@ -5,9 +5,10 @@ const exec = util.promisify(require('child_process').exec);
 describe('index e2e tests', function () {
 
     this.slow(5000);
+    this.timeout(5000);
```

Comment:
Good idea adding this. Maybe you should set it to 10 seconds? The 'slow' one can be kept at 5 so that we know it is taking longer than normal.